.github/workflows/update-proto.yml

```diff
@@ -3,6 +3,10 @@ name: "Update Proto"
 on:
   workflow_dispatch:
     inputs:
+      branch:
+        description: "Branch to update protos (default: master)"
```

Comment:
This confused me at first, I thought it may be the branch of the API repo we were updating but it's actually the branch of this repo we're pushing to. Unsure if clarification needed here.

Reply:
Updated description to be crystal clear. :)